src/types.ts

```diff
@@ -187,3 +187,7 @@ export type Invitation = {
   permission: string;
   name?: string;
 };
+
+export type Password = {
+  password: string;
+};
```

Comment:
Then wouldn't it be simple to have `Password = string` ? That's what I thought first without seeing the definition.